src/components/Medicine/AdministerMedicine.tsx

```diff
@@ -123,10 +124,10 @@ export default function AdministerMedicine({ prescription, onClose }: Props) {
 
   const { mutate: administerMedication } = useMutation({
     mutationFn: async (body: MedicationAdministration) =>
```

Comment:
just directly pass mutate fn here Instead of wrapping in a function

Reply:
We can't do that actually, we need the body param which is also passed inside the mutate fn.

Reply:
that's optional, we are doing that way in many other places

Reply:
See docs: https://docs.ohc.network/docs/care/development/data-fetching-in-care/#mutations

Example: https://github.com/ohcnetwork/care_fe/blob/3a9c7fea29169e2581d7b23238162c6e45bbf2e1/src/components/Schedule/ScheduleTemplateForm.tsx#L116-L120